src/uu/more/src/more.rs

```diff
@@ -52,9 +52,17 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
 
     // FixME: fail without panic for now; but `more` should work with no arguments (ie, for piped input)
     if let None | Some("-") = matches.value_of(options::FILE) {
-        println!("{}: incorrect usage", executable!());
+        show_error!("incorrect usage");
```

Comment:
You should probably use the `show_usage_error` macro here. It makes sure all tools have a uniform error message which is similar to GNU's.

And "incorrect usage" isn't a very descriptive message.

Reply:
```suggestion
        show_usage_error!("Reading from stdin isn't supported yet");
```